recipes/tsam/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "tsam" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/refs/heads/master.tar.gz
```

Comment:
We need a stable tag for this. We cannot package a dev release from `master`.

Reply:
A stable tag has been added in the lattest commit